src/CachePoolTest.php

```diff
@@ -343,7 +343,7 @@ public function testSaveExpired()
 
         $item = $this->cache->getItem('key');
         $item->set('value');
-        $item->expiresAt(new \DateTime('now'));
+        $item->expiresAt(\DateTime::createFromFormat('U', time()));
```

Comment:
using time() makes it possible to mock the clock, as done in Symfony's test suite